src/Features/Core/Portable/AddImport/References/ProjectSymbolReference.cs

```diff
@@ -36,7 +36,11 @@ protected override ImmutableArray<string> GetTags(Document document)
             {
                 return document.Project.Id == _project.Id
                     ? ImmutableArray<string>.Empty
-                    : WellKnownTagArrays.AddReference;
+                    : _project.Language == LanguageNames.CSharp
+                        ? WellKnownTagArrays.CSharpProject
```

Comment:
okay to leak this language specific info to core layer?

Reply:
in this case, yes.  it's not really doing anything more than saying: what is the project glyph for this project type.  given that this feature is really only about connection vb/c# projects, it's feels overly complex to have to introduce a language indirection.